doc/user-how-to/plugin-how-to.md

```diff
@@ -0,0 +1,46 @@
+# How to install and use QGreenland Custom
```

Comment:
This document needs a section on how to use the plugin's settings screen. I think it can just be moved from the other document.